tests/Valkey.Glide.UnitTests/ConnectionConfigurationTests.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
```

Comment:
Is there an integration test that can be done here, testing the outcome of this configuration?